content/administration/maintain/google_oauth.rst

```diff
@@ -0,0 +1,204 @@
+option will be located on the top left under the :guilabel:`Select a project` drop-down menu.
+
+On the :guilabel:`New Project` screen, rename the :guilabel:`Project name` to `Odoo` and browse for
+the :guilabel:`Location`. Set the :guilabel:`Location` as the `Google Workspace` organization. If
+you are using a personal Gmail account, then leave the :guilabel:`Location` as `No Organization`.
+
+.. image:: google_oauth/new-project.png
+   :class: img-fluid
+   :align: center
+   :alt: Project Name and Location for Google Oauth.
+
+Click on :guilabel:`Create` to finish this step. On the next screen, :guilabel:`User Type` options,
+select the :guilabel:`External` option, and then click on :guilabel:`Create` again, which will
+finally navigate to the :guilabel:`Edit app registration` page.
+
+Edit app registration
+---------------------
+
+- On the :guilabel:`Oauth consent screen` step, under the :guilabel:`App information` section,
+  enter `Odoo` in the :guilabel:`App name` field. Select the organization's email address under the
+  :guilabel:`User support` email field.
+- Next, under :menuselection:`App Domain --> Authorized domains`, click on :guilabel:`Add Domain`
+  and enter `odoo.com`.
+- After that, under the :guilabel:`Developer contact information` section, enter the organization's
+  email address. Google uses this email address to notify the organization about any changes to
+  your project.
+- Next, click on the :guilabel:`Save and Continue` button.
+- Then, skip the :guilabel:`Scopes` page by scrolling to the bottom and clicking on
+  :guilabel:`Save and Continue`.
+- Next, add the email being configured under the :guilabel:`Test users` step by clicking on
+  :guilabel:`Add Users` and then the :guilabel:`Save and Continue` button. A summary of the
+  :guilabel:`App registration` appears.
+- Finally, scroll to the bottom and click on :guilabel:`Back to Dashboard`.
+
+Create Credentials
+------------------
+
+Now that you have set up the project, you need to create credentials, which includes the
+:guilabel:`Client ID` and :guilabel:`Client Secret`. First, click on :guilabel:`Credentials` in the
+left sidebar menu.
+
+Then, click on :guilabel:`Create Credentials` in the top menu and select
+:guilabel:`OAuth client ID` from the dropdown menu.
+
+- Under :guilabel:`Application Type`, select :guilabel:`Web Application` from the dropdown menu.
+- In the :guilabel:`Client ID` field, enter `Odoo`.
+- Under the :guilabel:`Authorized redirect URIs` label, click the button :guilabel:`ADD URI`, and
+  then input `https://yourdbname.odoo.com/google_gmail/confirm` in the :guilabel:`URIs 1` field. Be
+  sure to replace the *yourdbname* part of the URL with your actual Odoo database name.
+- Next, click on :guilabel:`Create` to generate an OAuth :guilabel:`Client ID` and
+  :guilabel:`Client Secret`. Finally, copy each generated value to a Notepad and navigate to the
```

Comment:
Notepad as a proper noun assumes the user is using the Windows application.
Suggestion below, but not required.

```suggestion
  :guilabel:`Client Secret`. Finally, copy each generated value for later use when configuring in Odoo, and then navigate to the
```